modules/fun.js

```diff
@@ -0,0 +1,450 @@
+ * function slashFunAcronym
+ * @param {Discord.ChatInputCommandInteraction} int a /fun acronym interaction
+ */
+async function slashFunAcronym(int) {
+  const alphabet = [`A`, `B`, `C`, `D`, `E`, `F`, `G`, `H`, `I`, `J`, `K`, `L`, `M`, `N`, `O`, `P`, `Q`, `R`, `S`, `T`, `U`, `V`, `W`, `Y`, `Z`];
+  const len = int.options.getInteger(`length`) || Math.floor(Math.random() * 3) + 3;
+  const profanityFilter = require(`profanity-matcher`);
+  const pf = new profanityFilter();
+  let wordgen = [];
+
+  for (let ignored = 0; ignored < len * len; ignored++) {// try a bunch of times
+    for (let i = 0; i < len; i++) {
+      wordgen.push(alphabet[Math.floor(Math.random() * alphabet.length)]);
+    }
+    const word = wordgen.join(``);
+
+    if (pf.scan(word.toLowerCase()).length == 0) {
+      return int.editReply(`I've always wondered what __**${word}**__ stood for...`);
+    } else {
+      wordgen = [];
+    }
+  }
+  return int.editReply(`I ran into an error, try again and/or ping a botadmin.`);
+}
+
+/**
+ * function slashFunMinesweeper
+ * @param {Discord.ChatInputCommandInteraction} int a /fun minesweeper interaction
+ */
+async function slashFunMinesweeper(int) {
+  let size, mineCount;
+  // difficulty values
+  switch (int.options.getString(`difficulty`)) {
+    case `Hard`:
+      size = 14;
+      mineCount = 60;
+      break;
+    case `Medium`:
+      size = 10;
+      mineCount = 30;
+      break;
+    default:
+    case `Easy`:
+      size = 5;
+      mineCount = 5;
+      break;
+  }
+  // generate the minefield
+  let field = "";
+  // Getting all possible board spaces
+  const possibleSpaces = Array.from({ length: size * size }, (v, k) => k);
+  // Remove 4 corners, corners can't be mines
+  possibleSpaces.splice((size * size) - 1, 1);
+  possibleSpaces.splice((size - 1) * size, 1);
+  possibleSpaces.splice(size - 1, 1);
+  possibleSpaces.splice(0, 1);
+  // Finding out where the mines will be
+  const mineSpaces = [];
+  for (let i = 0; i < mineCount; i++) {
+    const random = Math.floor(Math.random() * possibleSpaces.length);
+    mineSpaces.push(possibleSpaces[random]);
+    possibleSpaces.splice(random, 1);
+  }
+
+  function getMineCount(x, y) {
+    let count = 0;
+    for (let i = -1; i <= 1; i++) {
+      if ((x + i) < 0 || (x + i) >= size) continue;
+      for (let j = -1; j <= 1; j++) {
+        if ((y + j) < 0 || (y + j) >= size) continue;
+        if (mineSpaces.includes((y + j) * size + x + i)) count++;
+      }
+    }
+
+    return count;
+  }
+
+  // Creating the final board
+  /** @type {number[][]} */
+  const board = [];
+  for (let x = 0; x < size; x++) {
+    board.push([]);
+    for (let y = 0; y < size; y++) {
+      if (mineSpaces.includes(x + (y * size))) {
+        board[x].push(9);
+        continue;
+      }
+      board[x].push(getMineCount(x, y));
+    }
+  }
+  const output = board.map(row => row.map(num => `||${num == 9 ? mineSweeperEmojis[`bomb`] : mineSweeperEmojis[num]}||`).join(``)).join(`\n`);
```

Comment:
The modification I talked about at the start of the file. Also includes a modification for the next comment
```suggestion
  const rows = board.map(row => row.map(num => `||${mineSweeperEmojis[num]}||`).join(``));
```